apps/stress-test/src/pages/v8/simple-stress/index.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { initializeIcons } from '@fluentui/react';
+import { StressApp } from '../../../components/v8/StressApp';
+
+initializeIcons();
+
+ReactDOM.render(<StressApp />, document.getElementById('root'));
```

Comment:
you could apply the same principle of creating generic test bench too. Why not just have `react/simple-stress` and `wc/simple-stress` ?

As I mentioned in a previous comment you can collect the actual implementations from webpack's `require.context` so you only need one page per stack. This makes adding different libraries on the same stack a lot easier